tab-bot-sso-with-backend/.vscode/tasks.json

```diff
@@ -209,36 +209,32 @@
             }
         },
         {
-            "id": "prepare-local-environment",
+            "id": "prepare-teams-app-dependencies",
             "type": "command",
-            "command": "fx-extension.debug-task-prepare-local-environment",
+            "command": "fx-extension.prepare-app-dependencies",
             "args": {
                 "enabledSteps": [
-                    "registerBot",
-                    "registerSSOAAD",
-                    "buildTeamsManifest",
+                    "registerBot", // Register Bot
+                    "registerSSOAAD", // Register AAD app for SSO
+                    "generateEnv", // Generate environment variables for local services
                 ],
-                "tab": {
-                    "port": 53000,
-                    "envSavePath": "${teamsfx:workspaceFolder}/tabs/.env.teamsfx.local"
-                },
-                "azureFunctions": {
-                    "port": 9229,
-                    "envSavePath": "${teamsfx:workspaceFolder}/api/.env.teamsfx.local"
-                },
-                "bot": {
-                    "port": 3978,
-                    "envSavePath": "${teamsfx:workspaceFolder}/bot/.env.teamsfx.local",
-                    "messagingEndpoint": "${teamsfx:botTunnelEndpoint}/api/messages"
+                "appConfigs": {
+                    "tabBaseUrl": "https://localhost:53000",
+                    "aadObjectId": "${teamsfx:aadObjectId}",
+                    "aadClientId": "${teamsfx:aadClientId}",
+                    "aadClientSecret": "${teamsfx:aadClientSecret}",
```

Comment:
We may provide different ways for customization:
- The most straightforward one - directly editing secret here
- Alternative to prevent credential in `tasks.json` - only editing IDs here, and editing secret in `.env` file

Just reminder for better doc.

Reply:
Nice reminder.